www/templates/main.html

```diff
@@ -5,42 +5,49 @@
 -->
 <ion-view>
 <ion-tabs ng-class="tabsCustomClass()">
-
-  <!-- Diary Tab -->
-  <ion-tab title="Diary" icon="ion-map" href="#/root/main/diary">
+  <!-- TODO: make a new leaderboards tab, and make a polar bear tab that shows if they're on emotion -->
```

Comment:
Can remove TODO!

Reply:
Also, you don't need to check in packages-lock.json and should use the icon font instead of separate icons to avoid duplication/extra space.